src/cms/views/dashboard/dashboard_view.py

```diff
@@ -36,4 +40,16 @@ def get(self, request, *args, **kwargs):
         """
 
         val = "To be defined"
-        return render(request, self.template_name, {"key": val, **self.base_context})
+        language_code = translation.get_language()
+        if language_code == "en-us":
+            feed = feedparser.parse("https://integreat-app.de/en/feed/")
+        elif language_code == "de-de":
+            feed = feedparser.parse("https://integreat-app.de/feed/")
```

Comment:
I propose to define a new setting in [src/backend/settings.py](https://github.com/Integreat/integreat-cms/blob/develop/src/backend/settings.py), e.g.
```
RSS_FEED_URLS = {
    "en-us": "https://integreat-app.de/en/feed/",
    "de-de": "https://integreat-app.de/feed/",
}
```
and use this like
```suggestion
        language_code = translation.get_language()
        feed = feedparser.parse(RSS_FEED_URLS[language_code])
```